cloudinary_cli/utils/api_utils.py

```diff
@@ -96,12 +98,34 @@ def handle_api_command(
 
     args, kwargs = parse_args_kwargs(func, params[1:]) if len(params) > 1 else ([], {})
 
+    if recursive and cursor is None:
+        raise Exception("Using the `--recursive` flag requires a `--cursor` flag.")
```

Comment:
"`--cursor` flag" is a bit confusing. Flag usually indicates whether something is on/off. I would use word `value` or `name` here.